superdesk/publish/publish_queue.py

```diff
@@ -142,9 +142,7 @@ def on_updated(self, updates, original):
                               )
 
     def _set_queue_state(self, updates, original):
-        destination = updates.get('destination', original.get('destination')) or {}
-        updates['state'] = QueueState.SUCCESS.value if destination.get('delivery_type') == 'content_api' \
-            else updates.get('state') or QueueState.PENDING.value
+        updates['state'] = updates.get('state') or QueueState.PENDING.value
```

Comment:
this looks like `updates.setdefault('state, QueueState.PENDING.value)`

Reply:
yes I've kept the way it was initially done, but a setdefault is better, or putting the value in second parameter of the get.